docs/getting-started/quickstart.js-backend.mdx

```diff
@@ -367,17 +387,22 @@ The [`Auth`](/docs/reference/backend/types/auth-object) object contains importan
 
         ```tsx {{ filename: 'src/api/example.ts' }}
         import { clerkClient } from '@clerk/astro/server'
+        import type { APIRoute } from 'astro'
 
-        export async function GET(context) {
-          // Use `locals.auth()` to access `isAuthenticated` and the user's ID
+        export const GET: APIRoute = async (context) => {
+          // The `Auth` object gives you access to properties like `isAuthenticated` and `userId`
+          // Accessing the `Auth` object differs depending on the SDK you're using
+          // https://clerk.com/docs/reference/backend/types/auth-object#how-to-access-the-auth-object
```

Comment:
I like the helpfulness of adding a link here, but they are going to go unchecked if/when we do updates or redirects. 

Because of that, should we leave them or remove them?